src/python/crash_analysis/severity_analyzer.py

```diff
@@ -104,8 +104,20 @@ def get_security_severity(crash_type, crash_output, job_name,
 class SeverityAnalyzerSanitizer(object):
   """Generic ASan severity analyzer."""
 
-  def analyze(self, crash_type, _, requires_gestures):
+  def analyze(self, crash_type, crash_output, requires_gestures):
     """Return a security severity based on the ASan crash output."""
+
+    manual_severity_match = re.search(
+        r'fuzzer-security-issue(-critical|-high|-medium|-low|)', crash_output,
+        re.IGNORECASE)
+    if manual_severity_match:
+      manual_severity = manual_severity_match.groups()[0].lower()
+      if manual_severity:
+        # Skip over the leading dash.
+        return string_to_severity(manual_severity[1:])
+      # Assign severity HIGH if none is specified explicitly.
+      return SecuritySeverity.HIGH
```

Comment:
Why don't you do this for low, critical, and medium severitys?

Reply:
Could you clarify your question? This line returns a default severity if no severity is specified via the marker. I chose HIGH because that is also the default in the UI.

Should this rather be a `MISSING` or just `None`?

Reply:
Let's use None here. We shouldn't try to guess the severity and instead encourage explicit severities in the string from the target.

Reply:
I simplified the design as per your other comment. As the severity has to be specified explicitly now, this special case no longer exists.